{"name": "针对异常登陆的测试", "status": "passed", "steps": [{"name": "步骤1：登录", "status": "passed", "start": 1609924903387, "stop": 1609924903389}], "attachments": [{"name": "log", "source": "f7f7992f-5bb4-41cd-a8c0-19353cd17cb0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'name': '登录功能-异常登录-用户名为空', 'username': '', 'password': '654321'}"}], "start": 1609924903383, "stop": 1609924903901, "uuid": "5748c71f-ff16-4e76-b4a6-4d6e5a7593c5", "historyId": "d93c8b521d1ee7fc4eb27338522fd359", "testCaseId": "130e6dce60bc4ef9d882c5f0465dc1d6", "fullName": "cases.test_login.TestLogin#test_login_failed", "labels": [{"name": "feature", "value": "金融界web登录模块"}, {"name": "story", "value": "登录模块-异常登陆"}, {"name": "epic", "value": "金融界web自动化自测"}, {"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PC-20200718ELAI"}, {"name": "thread", "value": "55088-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}